Keas.Mvc/ClientApp/components/Keys/KeyList.tsx

```diff
@@ -25,7 +26,7 @@ export default class KeyList extends React.Component<IProps, {}> {
                         <th/>
                         <th>Name</th>
                         <th>Code</th>
-                        <th className="">Available Serials</th>
+                        <th className=""><span id="serialTooltip">Available Serials<sup><i class="fas fa-info-circle"></i></sup></span><UncontrolledTooltip placement="bottom" target="serialTooltip">In Use / Total</UncontrolledTooltip></th>
```

Comment:
@jcubbage probably a little picky here but `sup` shouldn't be used for presentation, just for like exponents and such. Instead you'd use a `vertical-align: super` css property.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup